packages/react/src/components/Storage/FileUploader/FileUploader.tsx

```diff
@@ -34,6 +33,13 @@ export function FileUploader({
     Tracker = FileUploader.Tracker,
   } = components;
 
+  if (!acceptedFileTypes || !level) {
+    // eslint-disable-next-line no-console
+    console.warn(
```

Comment:
We have a logger util we can use for these message without having to disable linter.

```
import { Logger } from 'aws-amplify';
...
const logger = new Logger('FileUploader');
```

Reply:
Bonus to add a unit test that tests the logger is called in this curcumstance

Reply:
Sounds good! Updated in fbef484